### PR TITLE
contrib/pass-otp: new package (1.2.0)

### PR DESCRIPTION
**Pull request by jcgruenhage**

### CODE REVIEW

File: contrib/pass-otp/template.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+license = "GPL-3.0-or-later"
+url = "https://github.com/tadfisher/pass-otp"
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "5720a649267a240a4f7ba5a6445193481070049c1d08ba38b00d20fc551c3a67"
+# Has no test suite
+options = ["!check"]
+
+
+def do_build(self):
+    pass
```
Comment (jcgruenhage) [inline contrib/pass-otp/template.py:18]: Is this the right way to skip the build stage? This is just installing two files, one bash script (the pass extension itself) and one bash completion for pass with that extension

Comment (nekopsykose) [reply contrib/pass-otp/template.py:18]: yes

the other alternative is to not have a build style and manually call make install, but that is more clunky

Comment (nekopsykose) [reply contrib/pass-otp/template.py:18]: well, you can also not skip it? it doesn't look like it fails

Comment (nekopsykose) [reply contrib/pass-otp/template.py:18]: yeah it just prints some messages so you can let it do that :D

File: contrib/pass-otp/template.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+pkgname = "pass-otp"
+pkgver = "1.2.0"
+pkgrel = 0
+build_style = "makefile"
+hostmakedepends = []
```
Comment (nekopsykose) [inline contrib/pass-otp/template.py:5]:
```suggestion
```

File: contrib/pass-otp/patches/0001-fix-bash-completions-dir.patch [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+diff --git a/Makefile b/Makefile
+index a48cad1..185b446 100644
+--- a/Makefile
++++ b/Makefile
+@@ -4,7 +4,7 @@ DESTDIR ?=
+ LIBDIR ?= $(PREFIX)/lib
+ SYSTEM_EXTENSION_DIR ?= $(LIBDIR)/password-store/extensions
+ MANDIR ?= $(PREFIX)/share/man
+-BASHCOMPDIR ?= /etc/bash_completion.d
++BASHCOMPDIR ?= /usr/share/bash-completion/completions
```
Comment (nekopsykose) [inline contrib/pass-otp/patches/0001-fix-bash-completions-dir.patch:20]: you don't need a patch for this, you can pass it in make_install_args

File: contrib/pass-otp/template.py (hunk 1 above)
Comment (jcgruenhage) [reply contrib/pass-otp/template.py:18]: ¯\_(ツ)_/¯

**Pull request merged**